eahub/base/views.py

```diff
@@ -61,8 +61,8 @@ def getGroupsData():
         'map_data': map_data
     }
 
-def getProfilesData():
-    rows = Profile.objects.all()
+def getProfilesData(user):
+    rows = Profile.objects.visible_to_user(user)
     map_data = ''.join([
         '{' +
             'lat: {lat}, lng: {lon}, label:"{name}", path: "{path}", gdpr_confirmed: "{gdpr_confirmed}"'.format(
```

Comment:
as we're moving to "private", the gdpr_confirmed field in the map_data hash can be removed.